homeassistant/components/lamarzocco/binary_sensor.py

```diff
@@ -57,6 +58,16 @@ class LaMarzoccoBinarySensorEntityDescription(
     ),
 )
 
+SCALE_ENTITIES: tuple[LaMarzoccoBinarySensorEntityDescription, ...] = (
+    LaMarzoccoBinarySensorEntityDescription(
+        key="connected",
+        translation_key=None,
```

Comment:
why is it explicitly set to None?

Reply:
don't remember, I think I though it'd look more consistent